### DISCUSSION TITLE
URL non valido

### DISCUSSION THREAD

**Discussion by Giuseppleone** [community/community#41205] [Other Feature Feedback, Questions, & Ideas]
Quando faccio la registrazione non riesce a convalidare l'URL che devo fare?
![Screenshot_20221209-165829](https://user-images.githubusercontent.com/120198328/206742618-731d1824-f26a-4d4e-99e5-f9723f20e945.png)

**Comment by aconsuegra** [comment]
Hi @Giuseppleone, thanks for reaching out with your feedback! The "Mobile" category  is used only for discussions about the native mobile apps (Android and iOS), not for the mobile version of github.com. I will move this discussion to the "General" category, where I hope you can get some help.

**Comment by igorskyflyer** [comment]
Prova ad aggiungere https:// prima di www.
"https://www.unict.it/"
